GitUI/UserControls/RevisionGrid.cs

```diff
@@ -2695,37 +2696,29 @@ private void UpdateGraph(GitRevision rev)
 
         private void CheckUncommitedChanged(string filtredCurrentCheckout)
         {
+            bool enabled = AppSettings.UseFastChecks && !AppSettings.ShowGitStatusInBrowseToolbar && (FindForm() as FormBrowse)!=null;
```

Comment:
why do we need to check for FormBrowse?

Reply:
The count does only make sense for Browse, will otherwise show the total staged/unstaged also for FileHistory etc.
(this is slightly unrelated and could have been a separate issue but I feel I submit too many PRs already...)

Reply:
LOL you have indeed :)

Reply:
The small ones are easier to review though. 
I would have liked to present them as a series of commits instead, but then the review have to be done differently.
Maybe I will do next PR that way (the main remaining part of #4031), that is hard to split independently.

Reply:
I'm not suggesting to submit big PRs - changes should be atomic, self-contained and as small as possible.

The trouble with chained of commits (unless you submit them serially) is that if changes must be done to a parent, then all children must be rebased. And that can be very time consuming.